### PR TITLE
{bio}[foss/2018b,fosscuda/2018b] GROMACS v2018.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
GROMACS 2018.2 for foss and fosscuda 2018b

### CODE REVIEW

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4bdde8120c510b6543afb4b18f82551fddb11851f7edbd814aa24022c5d37857']
+
+builddependencies = [
+    ('CMake', '3.10.3'),
+]
+
+runtest = False
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb]: @akesandgren Why disable the tests?

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb]: @akesandgren `hpcugent` should be `easybuilders` (the tests should trip over this...)

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-fosscuda-2018b.eb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-fosscuda-2018b.eb]: `s/hpcugent/easybuilders/`

File: easybuild/easyconfigs/c/CMake/CMake-3.10.3-GCCcore-7.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.10.3'
```
Comment (boegel) [inline easybuild/easyconfigs/c/CMake/CMake-3.10.3-GCCcore-7.3.0.eb]: bump to most recent version `3.11.4`?

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb]: I forgot, I'll rebuild and let tests run...

File: easybuild/easyconfigs/c/CMake/CMake-3.10.3-GCCcore-7.3.0.eb (hunk 4 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CMake/CMake-3.10.3-GCCcore-7.3.0.eb]: We should do a last update, and then try and stick to `3.11.4` for the `2018b` generation of easyconfigs...

Comment (akesandgren) [reply easybuild/easyconfigs/c/CMake/CMake-3.10.3-GCCcore-7.3.0.eb]: Done the 3.11.4 upgrade etc.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
b-cn1522.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz, Python 2.7.12
See https://gist.github.com/c70627d31b1cbdc540284bf4a27afeac for a full test report.

**Comment by boegel** [comment]
@akesandgren Please remove the two `CMake` easyconfigs you're adding here. The older ones is just overhead, and the `3.11.4` one is already in `develop` (I overlooked that earlier)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/93c09c50f12dce01a62ebf5ad1504667 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3159.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c7144f06297c94b47102f3a4857e5649 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2104.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/31ca95665ad11b8fca29e169773a8db7 for a full test report.

**Pull request merged**